wcivf/apps/elections/templates/elections/postcode_view.html

```diff
@@ -3,86 +3,180 @@
+                        l.attr('aria-expanded', true);
+                        k.show()
+                    }
+                };
+                e.noSelect = function() {
+                    var h = 'none';
+                    return this.bind('selectstart dragstart mousedown', function() {
+                        return false
+                    }).css({
+                        MozUserSelect: h,
+                        msUserSelect: h,
+                        webkitUserSelect: h,
+                        userSelect: h
+                    })
+                };
+                if (g) {
+                    d = e.details = function() {
+                        return this.each(function() {
+                            var i = f(this),
+                            h = f('summary', i).first();
+                            h.attr({
+                                role: 'button',
+                                'aria-expanded': i.prop('open')
+                            }).on('click', function() {
+                                var j = i.prop('open');
+                                h.attr('aria-expanded', !j);
+                                i.triggerHandler((j ? 'close' : 'open') + '.details')
+                            })
+                        })
+                    };
+                    d.support = g
+                } else {
+                    d = e.details = function() {
+                        return this.each(function() {
+                            var h = f(this),
+                            j = f('summary', h).first(),
+                            i = h.children(':not(summary)'),
+                            k = h.contents(':not(summary)');
+                            if (!j.length) {
+                                j = f('<summary>').text('Details').prependTo(h)
+                            }
+                            if (i.length != k.length) {
+                                k.filter(function() {
+                                    return this.nodeType == 3 && /[^ \t\n\f\r]/.test(this.data)
+                                }).wrap('<span>');
+                                i = h.children(':not(summary)')
+                            }
+                            h.prop('open', typeof h.attr('open') == 'string');
+                            b(h, j, i);
+                            j.attr('role', 'button').noSelect().prop('tabIndex', 0).on('click', function() {
+                                j.focus();
+                                b(h, j, i, true)
+                            }).keyup(function(l) {
+                                if (32 == l.keyCode || (13 == l.keyCode && !c)) {
+                                    l.preventDefault();
+                                    j.click()
+                                }
+                            })
+                        })
+                    };
+                    d.support = g
+                }
+            }(document, jQuery));
```

Comment:
Wow formatter has made this go massive - I think we should put this whole block to how it was before to be sure it hasn't broken anything?